File: week-03/day-03/07.py
# create a function that takes a list and returns a new list with all the elements doubled

class Stuff():

    def __init__(self, *item):
        self.elements = []
        for a in item:
            self.elements.append(a)

    def double_elements(self):
        doubled = []
        for i in range(0, len(self.elements)):
            doubled.append(self.elements[i]*2)
        return doubled

list = Stuff(3, 5, 9, 6, 2)
print(list.double_elements())
list1 = Stuff(5, 34, 9, 21)
print(list1.double_elements())

File: week-07/day-03/07.js
'use strict';

var numbers = [2, 5, 11, 29];

function isPrime(number) {
  if (number > 1) {
    for (var i = 2; i < number; i++) {
      if (number % i === 0) {
        return false;
      }
    }
  }
  return true;
}

function allPrime(arr) {
  console.log(arr.every(isPrime));
}

allPrime(numbers);

// create a function that takes an array of numbers and returns a boolean
// it should return true if all the elements are prime, false otherwise
